hudi-common/src/test/java/org/apache/hudi/common/table/timeline/versioning/BaseTestCommitMetadataSerDe.java

```diff
@@ -72,7 +85,7 @@ protected void testEmptyMetadataSerDe() throws Exception {
     assertTrue(serialized.isPresent());
 
     // Deserialize
-    HoodieCommitMetadata deserialized = serDe.deserialize(instant, serialized.get(), HoodieCommitMetadata.class);
+    HoodieCommitMetadata deserialized = serDe.deserialize(instant, new ByteArrayInputStream(serialized.get()), () -> true, HoodieCommitMetadata.class);
```

Comment:
Similar here.  Revisit all `() -> true` to see if they should be `() -> false` (non-empty file).